scripts/release/atomic_pipeline.py

```diff
@@ -126,7 +126,7 @@ def build_meko_tests_image(build_configuration: ImageBuildConfiguration):
     shutil.copyfile("release.json", "docker/mongodb-kubernetes-tests/release.json")
     shutil.copyfile("requirements.txt", requirements_dest)
 
-    python_version = os.getenv("PYTHON_VERSION", "3.13")
+    python_version = os.getenv("PYTHON_VERSION", "3.13.7")
```

Comment:
I think it's better to fail here, if the PYTHON_VERSION is not set. Otherwise we would need to remember to update it in multiple places